src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -35,11 +35,7 @@
 import org.terasology.world.block.BlockUri;
 
 import java.math.RoundingMode;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.Timer;
-import java.util.TimerTask;
+import java.util.*;
```

Comment:
No star imports. Please revert this.

I would recommend you set the limit for star imports in IntelliJ to a very high value, like 999.

(search around on google for how to do this).